app/Http/Controllers/Admin/NewsletterCrudController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+
+// VALIDATION: change the requests to match your own file names if you need form validation
+use App\Http\Requests\NewsletterRequest as StoreRequest;
+use App\Http\Requests\NewsletterRequest as UpdateRequest;
+
+/**
+ * Class NewsletterCrudController
+ * @package App\Http\Controllers\Admin
+ * @property-read CrudPanel $crud
+ * @codeCoverageIgnore
+ */
+class NewsletterCrudController extends CrudController
+{
+    public function setup()
+    {
+        /*
+        |--------------------------------------------------------------------------
+        | CrudPanel Basic Information
+        |--------------------------------------------------------------------------
+        */
+        $this->crud->setModel('App\Models\Newsletter');
+        $this->crud->setRoute(config('backpack.base.route_prefix') . '/newsletter');
+        $this->crud->setEntityNameStrings('newsletter', 'newsletters');
+
+        /*
+        |--------------------------------------------------------------------------
+        | CrudPanel Configuration
+        |--------------------------------------------------------------------------
+        */
+
+        $this->crud->addField([
+            'name'=>'email',
+            'label'=>'Email',
+            'type'=>'email',
+        ]);
+        
+        $this->crud->addField([
+            'name'=>'token',
+            'label'=>'Token',
+            'type'=>'email',
```

Comment:
I think this shouldn't be email type, because if I try to hover over the token field I get this:
![screenshot from 2018-09-08 13-24-20](https://user-images.githubusercontent.com/14909670/45253623-9d31ec00-b36a-11e8-8553-05d04edbbfee.png)